src/types/change.ts

```diff
@@ -41,6 +42,7 @@ type InsertDeleteAttrs = {
   statusUpdateAt: number
   createdAt: number
   updatedAt: number
+  movedChangeId?: string
```

Comment:
If you would create a __nodeMove__ change in _trackTransaction_ you can achieve higher consistency here by creating attributes for CHANGE_OPERATION.move and not extending InsertDelete with an optional property. This flag converts this attributes into CHANGE_OPERATION.move anyway but creating it explicitly with { operation: CHANGE_OPERATION.move } will be clearer. Also, it is possible to have moved and deleted node at the same time? And how what that work?